### PR TITLE
Cleanup of English translation file

### PR DESCRIPTION
**Pull request by harrieklomp**
Some cleanup of English translation file. Please re-check and correct if needed!

### CODE REVIEW

File: lib/languages/en.php [hunk 1]
```diff
@@ -2643,7 +2468,6 @@
     'admin_notes_14' => 'Notifications about logs in the watched caches have been disabled',
     'admin_notes_15' => 'User has been activated',
     'removingLog' => 'removing log entry',
-    'wayPointType-1' => 'Select one waypoint',
```
[the remark this answers is not in the document]
Comment (harrieklomp) [reply lib/languages/en.php]: viewPending_08 should be in English translation file please look at #1801

File: lib/languages/en.php [hunk 2]
```diff
@@ -2383,9 +2214,7 @@
     'removedLog_01' => 'has removed your log entry for the cache',
     'removedLog_02' => 'Contents of the log was',
     'notify_subject' => 'New cache on {site_name}',
-    'notify_subject_pl' => 'New caches on {site_name}',
     'notify_intro' => 'Good news! A new cache was placed in your area',
-    'notify_intro_pl' => 'Good news! New caches were placed in your area',
```
[the remark this answers is not in the document]
Comment (deg-pl) [reply lib/languages/en.php]: Yes! Keep it please.

**Comment by harrieklomp** [comment]
Done some corrections.
Do not know witch 'register_confirm' to remove?

**Comment by following5** [comment]
The first 'register_confirm' is used in the obsolete _old_menu. So I think that can be removed.

**Comment by harrieklomp** [comment]
For now it is ready for merge.

**Pull request merged**